Orange/data/sql/table.py

```diff
@@ -355,7 +355,7 @@ def _compute_basic_stats(self, columns=None,
         if columns is not None:
             columns = [self.domain[col] for col in columns]
         else:
-            columns = list(self.domain)
+            columns = list(self.domain.variables)
```

Comment:
Remove casting to `list` here and below?

Reply:
wouldn't this break `columns += list(self.domain.metas)` in line 360?

Reply:
I wrote "here and below". :)